app/models/manageiq/providers/redhat/infra_manager/vm_import.rb

```diff
@@ -60,9 +84,14 @@ def perform_vmware_to_ovirt_import(params)
           :sparse         => params[:sparse],
         )
       )
+      ems_ref_for_vm_id(import.vm.id)
     end
   end
 
+  def ems_ref_for_vm_id(vm_id)
+    "/api/vms/#{vm_id}"
```

Comment:
We should always avoid building paths like this. We had (and will have) many issues because of changing or incorrect paths. As in the previous step you already got a VM object, which should have an `href` attribute, the right way to calculate the EMS reference is to use the `make_ems_ref` method, defined here:

https://github.com/ManageIQ/manageiq-providers-ovirt/blob/master/app/models/manageiq/providers/redhat/infra_manager/api_integration.rb#L340-L345

That `make_ems_ref` should be the only place where we do this kind of path manipulation.

To use it you need something like this:

```ruby
ManageIQ::Providers::Redhat::InfraManager.make_ems_ref(import.vm.href)
```